app/controllers/api/v1/samples_controller.rb

```diff
@@ -1,4 +1,4 @@
-class Api::SamplesController < ApplicationController
+class Api::V1::SamplesController < ApplicationController
```

Comment:
This is good, but we need to update `config/routes.rb` file as well with the new namespace.